Xamarin.Forms.Core/RefreshView.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Windows.Input;
+using Xamarin.Forms.Platform;
+
+namespace Xamarin.Forms
+{
+	[ContentProperty("Content")]
+	[RenderWith(typeof(_RefreshViewRenderer))]
+	public class RefreshView : ContentView, IElementConfiguration<RefreshView>
+	{
+		readonly Lazy<PlatformConfigurationRegistry<RefreshView>> _platformConfigurationRegistry;
+
+		public RefreshView()
+        {
+            IsClippedToBounds = true;
+            VerticalOptions = LayoutOptions.FillAndExpand;
+            HorizontalOptions = LayoutOptions.FillAndExpand;
+			
+			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<RefreshView>>(() => new PlatformConfigurationRegistry<RefreshView>(this));
+		}
+        
+        public static readonly BindableProperty IsRefreshingProperty =
+            BindableProperty.Create(nameof(IsRefreshing), typeof(bool), typeof(RefreshView), false);
+			
+        public bool IsRefreshing
+        {
+            get { return (bool)GetValue(IsRefreshingProperty); }
+            set
+            {
+                if ((bool)GetValue(IsRefreshingProperty) == value)
+                    OnPropertyChanged(nameof(IsRefreshing));
```

Comment:
A bit confused with this line. Shouldn't we call `OnPropertyChanged` only after the property changes? 

```
var oldValue = IsRefreshing;

SetValue(IsRefreshingProperty, value);

if(oldValue != IsRefreshing)
   OnPropertyChanged(nameof(IsRefreshing));
```

Also, doesn't a bindable property already have change notification built in? Why use `OnPropertyChanged` again?

Reply:
This is not necessary. The BindableProperty already notifies the change.